test/screenings.spec.js

```diff
@@ -0,0 +1,53 @@
+import {
+    describe,
+    expect,
+    test
+  } from "@jest/globals"
+import { filterOutOldScreenings } from "../src/api.js";
+
+
+
+
+//mock time
+const fakeToday = new Date("2023-02-05T21:00:00.000Z");
+
+//TODO make mock in json format
+
+const items = { 
+    data: [
+            {
```

Comment:
This block of code is inconsistently indented compared to other parts of this file and others.